examples/private/version.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 1.3.0, <1.7.0"
+  required_providers {
+    ibm = {
+      source  = "IBM-Cloud/ibm"
+      version = "1.62.0"
```

Comment:
The default example is already running the lowest supported IBM provider version, so this example should run on latest. So change this to `>= 1.62.0`